GitUI/Editor/FileViewer.cs

```diff
@@ -666,29 +666,27 @@ private string GetFileText(string fileName)
                 path = _fullPathResolver.Resolve(fileName);
             }
 
-            if (File.Exists(path))
+            if (!File.Exists(path))
             {
-                // StreamReader disposes of 'fileStream'.
-                // see: https://msdn.microsoft.com/library/ms182334.aspx
-                var fileStream = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
-                try
-                {
-                    using (var reader = new StreamReader(fileStream, Module.FilesEncoding))
-                    {
-                        fileStream = null;
-                        var content = reader.ReadToEnd();
-                        FilePreamble = reader.CurrentEncoding.GetPreamble();
-                        return content;
-                    }
-                }
-                finally
+                return null;
+            }
+
+            // StreamReader disposes of 'fileStream'.
+            // see: https://msdn.microsoft.com/library/ms182334.aspx
+            var fileStream = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+            try
+            {
+                using (var reader = new StreamReader(fileStream, Module.FilesEncoding))
                 {
-                    fileStream?.Dispose();
+                    fileStream = null;
+                    var content = reader.ReadToEnd();
+                    FilePreamble = reader.CurrentEncoding.GetPreamble();
+                    return content;
                 }
             }
-            else
+            finally
             {
-                return null;
+                fileStream?.Dispose();
```

Comment:
a separate piece of work - to replace `try/finally` with `using`

Reply:
Done in a4f62fed98a5d6bd9ae6fc223585f79247b8a66f.